### PR TITLE
Search page implementation

### PR DESCRIPTION
**Pull request by SteveOscar**
This sets up the welcome screen/search page for the app.  
- The FMPicker component is creates the list of countries that users can choose from.  
- The SearchPage component is the home screen, which renders the FMPicker,  buttons, title and graphics.  
- The Index component handles the pseudo navigation/routing of the app.

### CODE REVIEW

**Comment by PenneyGadget** [comment]
@SteveOscar I don't know much about React Native, but this looks good and it was interesting to see your setup. I also really appreciate your PR message as it very clearly outlines what's happening here.
Only question/concern: what are all those lines in the FMPicker highlighted bright red as though from a linter? I haven't seen that on github before...

**Comment by brantwellman** [comment]
@SteveOscar, unfortunately I also don't know much about RN, however I noticed that you have a few lines of commented code in your index.ios.js and SearchPage.js files that you might want to consider removing before merging.

File: DropDown.js [hunk 1]
```diff
@@ -1,32 +1,22 @@
...
-      <Picker.Item label="Ruby" value="ruby" />
-      <Picker.Item label="wow" value="wow" />
-      <Picker.Item label="sc2" value="sc2" />
-      <Picker.Item label="Java" value="java" />
-      <Picker.Item label="JavaScript" value="js" />
-    </Picker>
-  )
+      <Picker>
+        <Picker.Item label="Ruby" value="ruby" />
+        <Picker.Item label="wow" value="wow" />
```
Comment (rrgayhart) [inline DropDown.js:13]: Wow!

File: Expenses.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      {rowHasChanged: (r1, r2) => r1[0] !== r2[0]});
+    this.state = {
+      dataSource: dataSource.cloneWithRows(this.props.listings)
+    };
+  }
+
+  renderRow(rowData, sectionID, rowID) {
+    return (
+      <TouchableHighlight
+          underlayColor='#dddddd'>
```
Comment (rrgayhart) [inline Expenses.js:26]: Should this also be int he stylesheet?

Comment (SteveOscar) [inline Expenses.js:26]: @rrgayhart yep I'll move the inline styling to the proper style regions. The React Native tutorials I've seen all have style objects in-file that are referenced. Still haven't figured out if this is standard or sloppy.

**Comment by SteveOscar** [comment]
@PenneyGadget I'm not sure either why those lines are so severely highlighted red. Looking into it!

**Comment by SteveOscar** [comment]
@brantwellman thanks for pointing that out.

**Pull request merged**